src/test/videopress/edit.js

```diff
@@ -44,5 +55,116 @@ describe( 'VideoPress block', () => {
 
 		const expectedHtml = `<!-- wp:videopress/video /-->`;
 		expect( getEditorHtml() ).toBe( expectedHtml );
+
+		// Reset editor to avoid side effects from built-in timers in addBlock() function on iOS
+		await initializeEditor();
```

Comment:
@fluiddot, if you comment out this line and run `TEST_RN_PLATFORM=ios npm run test src/test/videopress/edit.js`, the first test in the second test suite (no matter what that test is) fails with the following:

```
    TypeError: Cannot read property 'presentPicker' of null

      195 | 			if ( autoOpen && isIOS ) {
      196 | 				this.pickerTimeout = setTimeout(
    > 197 | 					() => this.picker.presentPicker(),
          | 					                  ^
      198 | 					PICKER_OPENING_DELAY
      199 | 				);
      200 | 			} else {
```

If you navigate to that code directly and change the conditional so that `this.picker.presentPicker` runs without a delay, the test passes. Similarly, the test will also pass after commenting out the `addBlock` call, which makes me think that the failure is related to timing/delays, leading to this hack.

The PR related to the issue you linked to, https://github.com/WordPress/gutenberg/commit/023631c27cfecd9684d85a4bf92570158660edbf, doesn't resolve the issue unfortunately.

Reply:
Ah, wow, thank you Carlos, such a seemingly simple solution to something I was racking my brain over. 🤦‍♀️ I've gone ahead to apply your fix in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5751/commits/d56e303a6ab9570e5acd897aec9f06c8725af4c7.